daemon/logger/omslogs/omslogs_test.go

```diff
@@ -0,0 +1,123 @@
+package omslogs
+
+import (
+	"testing"
+
+	"bytes"
+	"fmt"
+	"github.com/docker/docker/daemon/logger"
+)
```

Comment:
To fix the CI failure on this file, change to;

```go
import (
	"bytes"
	"fmt"
	"testing"

	"github.com/docker/docker/daemon/logger"
)
```

Reply:
Thanks for the help. I've made the corrects and pushed the change.